outputs.tf

```diff
@@ -1,29 +1,11 @@
-output "application_endpoint" {
-  value       = "https://${var.frontend_dns}.${substr(data.google_dns_managed_zone.dnszone.dns_name, 0, length(data.google_dns_managed_zone.dnszone.dns_name) - 1)}"
-  description = "The URI to access the Terraform Enterprise Application."
+output "application_url" {
+  value = "https://${module.dns.fqdn}"
 }
 
-output "application_health_check" {
-  value       = "https://${var.frontend_dns}.${substr(data.google_dns_managed_zone.dnszone.dns_name, 0, length(data.google_dns_managed_zone.dnszone.dns_name) - 1)}/_health_check"
-  description = "The URI for the Terraform Enterprise Application health check."
+output "dashboard_url" {
+  value = module.cluster.installer_dashboard_url
```

Comment:
Quick question: We use the FQDN for `application_url` but `dashboard_url` uses the IP, is this because `module.cluster` has no sense of what module `dns` is up to here? (which makes sense as they could be used separately). 

Wondering if we should use the `fqdn` here too to avoid cert errors on the IP. It'd be an educated guess of sorts since technically what we have here is the most correct but `"https://${module.dns.fqdn}:8800"` will almost always be the right thing (trying to think of what would be a situation where it's not). 

tbh kinda back and forth on it myself but wanted to bring it up.